src/Run/CommandLine.cs

```diff
@@ -762,8 +762,7 @@ public string GetHelp(int maxLineWidth, string parameterSetName = null, bool dis
                     }
                 }
             }
-
-            if (hasQualifiers)
+            else if (hasQualifiers)
```

Comment:
@maririos please keep in mind that you want to merge this back with the other CommandLine.cs file at some point so lets try not to make the assumption that it is only used by the run tool. As for the actual issue here I didn't dig in so I'm trusting your understanding that the are mutually exclusive.